docs/Installation_Alternative.rst

```diff
@@ -36,13 +36,13 @@ you can use one of the following simple commands:
 
 -  using ``easy_install`` (old tool, but still works)::
 
-       easy_install --prefix /tmp easybuild
+       easy_install --prefix /tmp/$USER easybuild
```

Comment:
I don't think there is a sensible default... I'd like to stay away from suggesting `$HOME`, that's too much of a can of worms with Python (think `--user` which installs in the magic path `$HOME/.local`).

`/tmp/$USER` already beats `/tmp` out of the park, imho